mikado-app/src/pages/Graph/MyDrawer.js

```diff
@@ -23,6 +23,9 @@ function MyDrawer({ displayLayerHandle }) {
   >
     <div id="puller"></div>
     {drawerToggle && <TextField id="filled-basic" label="Node Name" variant="filled" defaultValue={selectedNodeName} onChange={e => displayLayerHandle.setSelectedNodeName(e.target.value)} />}
+    <Button onClick={drawerButtonClick}>
+      Create Subgraph
```

Comment:
TODO: I will disable this and other things. This is because the database schema changed in the meantime, and it would complicate things. While this would have normally added complexity to the database migration, we are doing that manually if ever, and it still adds complexity if a subgraph comes into existence in the DB. Also, we want to only enable this feature once it works in its entirety. I also plan to disable other WIP features, but keep the code to build off of.